lua/telescope/builtin/git.lua

```diff
@@ -195,7 +195,14 @@ git.branches = function(opts)
     .. "%(authorname)"
     .. "%(upstream:lstrip=2)"
     .. "%(committerdate:format-local:%Y/%m/%d %H:%M:%S)"
-  local output = utils.get_os_command_output({ "git", "for-each-ref", "--perl", "--format", format }, opts.cwd)
+
+  local cmd = { "git", "for-each-ref", "--perl", "--format", format }
```

Comment:
Nitty remark, this can be simplified as follows.

```lua
 local output = utils.get_os_command_output({  "git", "for-each-ref", "--perl", "--format", format, opts.pattern }, opts.cwd)
```

since lua does not insert `nil` into tables, e.g.`print(vim.inspect({1, nil}))` which prints `{1}`

Reply:
I actually tried that but I was getting an error when I didn't provide a pattern. I will double check!

Reply:
Ok :) maybe it doesn't work for some reason in this context? Not sure

Reply:
No you were right, just tried it. I pushed up a change. idk